thirdparty/runtime_environments.go

```diff
@@ -507,17 +467,43 @@ func (c *RuntimeEnvironmentsClient) GetEvents(ctx context.Context, opts EventHis
 			}
 			entries = append(entries, *entry)
 		}
-		timestamp, err := stringToTime(imageHistory[i].CreationDate)
-		if err != nil {
-			return nil, errors.Wrap(err, "converting creation date")
-		}
+		timestamp := time.Unix(int64(imageHistory[i].CreationDate), 0)
 		imageEvent := ImageEvent{
 			Entries:   entries,
 			Timestamp: timestamp,
 			AMIBefore: amiBefore,
-			AMIAfter:  imageHistory[i].AMI,
+			AMIAfter:  amiAfter,
 		}
 		result = append(result, imageEvent)
 	}
 	return result, nil
 }
+
+// Image stores information about an image including its AMI, ID, and last deployed time.
+type Image struct {
```

Comment:
we use image all over the codebase, so i worry this isn't specific enough. Could we call the struct DistroImage or something, to give it context?